app/services/CheckoutService.scala

```diff
@@ -102,8 +106,7 @@ class CheckoutService(identityService: IdentityService,
       case -\/(errSeq) =>
         \/.left(errSeq.<::(CheckoutIdentityFailure(
           s"Could not update identity details for subscribed user ${authenticatedUser.user.id}",
-          memberId.toString,
-          None)))
+          Some(s"SF Account ID = ${memberId.salesforceAccountId}, SF Contact ID = ${memberId.salesforceContactId}"))))
```

Comment:
`memberId.toString` currently returns useless message, so `ContactId.toString`  should be improved in `membership-common`. Until then using overly verbose `${memberId.salesforceAccountId}, SF Contact ID = ${memberId.salesforceContactId}"` to get some useful info.